funcx_endpoint/funcx_endpoint/strategies/kube_simple.py

```diff
@@ -45,99 +44,68 @@ def strategize(self, *args, **kwargs):
             pass
 
     def _strategize(self, *args, **kwargs):
-        task_breakdown = self.interchange.get_outstanding_breakdown()
-        logger.debug(f"Task breakdown {task_breakdown}")
+        max_pods = self.interchange.config.provider.max_blocks
+        min_pods = self.interchange.config.provider.min_blocks
 
-        min_blocks = self.interchange.config.provider.min_blocks
-        max_blocks = self.interchange.config.provider.max_blocks
+        # Kubernetes provider doesn't really know how to keep track of pods by container
+        # so best to allow it to shut them all down when quiet
+        assert min_pods == 0
 
-        # Here we assume that each node has atleast 4 workers
+        # Kubernetes provider only supports one manager in a pod
+        managers_per_pod = 1
 
-        tasks_per_node = self.interchange.config.max_workers_per_node
-        if self.interchange.config.max_workers_per_node == float('inf'):
-            tasks_per_node = 1
+        tasks_per_pod = self.interchange.config.max_workers_per_node
```

Comment:
`workers_per_pod` is more accurate?

Reply:
Yes - thank you. Hopefully there will be several tasks run on that worker!